src/tests/sqlsmith/src/runner.rs

```diff
@@ -27,6 +30,106 @@ use crate::{
     sql_gen, Table,
 };
 
+/// e2e test runner for pre-generated queries from sqlsmith
+pub async fn run_pre_generated(client: &tokio_postgres::Client, outdir: &str) {
+    let ddl_path = format!("{}/ddl.sql", outdir);
+    let queries_path = format!("{}/queries.sql", outdir);
+    let ddl = std::fs::read_to_string(ddl_path).unwrap();
+    let queries = std::fs::read_to_string(queries_path).unwrap();
+    let mut setup_sql = String::with_capacity(1000);
+    for ddl_statement in parse_sql(&ddl) {
+        let sql = ddl_statement.to_string();
+        tracing::info!("Executing: {}", sql);
+        let response = client.execute(&sql, &[]).await;
+        if let Err(e) = response {
+            panic!("{}", format_fail_reason(&setup_sql, &sql, &e))
+        }
+        setup_sql.push_str(&sql);
+    }
+    for statement in parse_sql(&queries) {
+        let sql = statement.to_string();
+        tracing::info!("Executing: {}", sql);
+        let response = client.query(&sql, &[]).await;
+        if let Err(e) = response {
+            panic!("{}", format_fail_reason(&setup_sql, &sql, &e))
+        }
+    }
+}
+
+/// e2e query generator
+/// The goal is to generate NON-FAILING queries.
```

Comment:
If we can support ignoring cases, this seems not necessary any more. In my mind, we can generate cases regardless of success of failure, and then mark to ignore thse unsupported ones.

Reply:
Agree, plan to work on this in a separate PR.